src/main/java/org/wikivoyage/listings/language/Language.java

```diff
@@ -29,12 +29,19 @@
     String getNameElement();
     
     /**
+     * The name of the place, which is the only strictly required element.
+     */
+    String getFlagElement();
+    
+    /**
      * Convert listing template into a WikivoyagePOI object.
+     * @param string 
```

Comment:
I suggest listing the possible values of poiType and their meaning here, so that language developers can find out quickly what it means. Also, I suggest using `null` rather than `""` for the default value.